drivers/comparator/Kconfig.stm32_comp

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config COMPARATOR_STM32_COMP
+	bool "ST STM32 comparator driver"
+	default y
+	depends on DT_HAS_ST_STM32_COMP_ENABLED
+	select PINCTRL
+	select USE_STM32_LL_COMP
+	select USE_STM32_LL_EXTI
+
+config COMPARATOR_STM32_COMP_EXTI_HAS_LINES_64_95
+	bool "ST STM32 EXTI has lines 64 to 95"
```

Comment:
If this is soc specific and user has no configuration to do, please remove the prompt ("ST [...]"), so user cannot change it

Reply:
Changed to: "STM32 EXTI has lines 64 to 95"